activemodel/test/cases/attribute_test.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require "cases/helper"
+require "active_model/attribute"
+require "active_model/type"
```

Comment:
Since `Type.default_value` is used in `Attribute::Null`, I think that the `require "active_model/type"` should be included in `active_model/attribute`.

Reply:
Seems reasonable. I updated. Thanks!